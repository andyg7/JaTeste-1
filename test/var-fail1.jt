func int main()
{

	return 0;
}

struct phone {
	int price;
	int model;
	int year;
	bool iphone;

	method void set_iphone(bool b)
	{
		phone = b;
	}
};
