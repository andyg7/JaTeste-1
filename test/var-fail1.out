Scanned
Parsed
Fatal error: exception Exceptions.UndeclaredVariable("phone")
